### PR TITLE
Create 2023-04-09-GDAC.md

### PR DESCRIPTION
**Pull request by svg-arch**

### CODE REVIEW

File: content/attacks/posts/2023-04-09-GDAC.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+GDAC lost approximately $13 million:
+- 10,000,000 WEMIX
+- 220,000 USDT
+- 350 ETH
+- 60.8 BTC
+
+## Timeline
+
+- **April 08, 2023, 06:36:59 PM +UTC:** [The exploiter starts to attack](https://etherscan.io/tx/0x242fea7585152bc1652aac4dbe9db3e8ceaa428f5372e22d40071ee2b50733a7)
+- **April 10, 2023:** [GDAC announced the hack on their platform.](https://support.gdac.com/hc/ko/articles/17389345782041--%EA%B8%B4%EA%B8%89%EA%B3%B5%EC%A7%80-%EC%9E%85%EC%B6%9C%EA%B8%88-%EC%84%9C%EB%B9%84%EC%8A%A4-%EC%9D%BC%EC%8B%9C%EC%A4%91%EB%8B%A8-%EB%B0%8F-%EA%B8%B4%EA%B8%89-%EC%84%9C%EB%B2%84%EC%A0%90%EA%B2%80-%EC%A7%84%ED%96%89-%EC%82%AC%EC%9C%A0)
```
Comment (JediFaust) [inline content/attacks/posts/2023-04-09-GDAC.md:38]: Fix source link please

File: content/attacks/posts/2023-04-09-GDAC.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+## Attackers
+
+The attackers behind the GDAC hack remain unidentified.
+
+GDAC Hacker ETH wallets:
+- [0x244615D99684175d31369332039b2D84ce925EC5](https://etherscan.io/address/0x244615D99684175d31369332039b2D84ce925EC5)
+- [0x62B5eb2cb925Ce2898f9327B235b3228e7Cac1C2](https://etherscan.io/address/0x62B5eb2cb925Ce2898f9327B235b3228e7Cac1C2)
+- [0x87597bDB421482190e223aCa0A4DEAd75AB0a98D](https://etherscan.io/address/0x87597bDB421482190e223aCa0A4DEAd75AB0a98D)
+- [0x6225c189f583fc2e17f5be36a6a9db4d3978872d](https://etherscan.io/address/0x6225c189f583fc2e17f5be36a6a9db4d3978872d)
+- [0x57192cca8b8e4beb77f3466c6d0550e64cc53b0f](https://etherscan.io/address/0x57192cca8b8e4beb77f3466c6d0550e64cc53b0f)
```
Comment (JediFaust) [inline content/attacks/posts/2023-04-09-GDAC.md:25]: As usual, hackers utilizes a lot of addresses, please mention few of them or even single one, if there is only one marked on etherscan

**Review by JediFaust — CHANGES_REQUESTED** [review]

File: content/attacks/posts/2023-04-09-GDAC.md (hunk 1 above)
Comment (svg-arch) [reply content/attacks/posts/2023-04-09-GDAC.md:38]: There is only a Korean article and the link contains Korean letters

Comment (JediFaust) [reply content/attacks/posts/2023-04-09-GDAC.md:35]: I tried remove the letters after digits like this: [https://support.gdac.com/hc/ko/articles/17389345782041](https://support.gdac.com/hc/ko/articles/17389345782041) and it still refers to the same page, so it's unnecessary

**Review by JediFaust — CHANGES_REQUESTED** [review]

**Review by JediFaust — APPROVED** [review]

**Pull request merged**